python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -45,6 +47,11 @@
 MOCK_CTRL_START_TIME = 20
 TELEMETRY_START_TIME = 30
 
+# Maximum time to wait for rotator telemetry (seconds).
+# Must be less than the maximum time the CCW waits for a tracking command
+# before going to FAULT.
+ROTATOR_TELEMETRY_TIMEOUT = 1
```

Comment:
Which parameters defines the "maximum time the CCW waits for a tracking command before going to FAULT"? Can you add in the comment here?

Reply:
That is an internal detail of the low-level controller and I don't know how it is configured.